docs/writing-policies/spec/host-capabilities/05-crypto.md

```diff
@@ -28,18 +28,29 @@ performing cryptographic checks exposed by the host:
 <td>
 
 ```hcl
+# Certificate:
+{
+  # **mandatory**: Which encoding is used by the certificate
+  # Either the string "Pem" or "Der"; case sensetive.
```

Comment:
Is this really case sensitive? I think we should change the policy-evaluator code to make it case insensitive.
If you agree with that, let's open an issue to track that

Reply:
I agree. I created an issue and a PR to solve it.

Issue: https://github.com/kubewarden/policy-sdk-rust/issues/93
PR: https://github.com/kubewarden/policy-sdk-rust/pull/94

CC: @flavio @jvanz

Reply:
linked PR looks good to me, approving this.